### PR TITLE
Pavan/distro detect

### PR DESCRIPTION
**Pull request by pasankav**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/connectedk8s/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.3.0
```
Comment (akashkeshari) [inline src/connectedk8s/HISTORY.rst]: Please change it to 0.2.1 as it is a minor change

File: src/connectedk8s/setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.2.0'
+VERSION = '0.3.0'
```
Comment (akashkeshari) [inline src/connectedk8s/setup.py]: 0.2.1

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 3]
```diff
@@ -454,6 +456,20 @@ def get_server_version(configuration):
                                 summary='Unable to fetch kubernetes version')
         logger.warning("Unable to fetch kubernetes version: %s\n", e)
```
Comment (akashkeshari) [inline src/connectedk8s/azext_connectedk8s/custom.py:459]: Please add another newline here. There is a consistency of 2 line gaps between functions

Comment (pasankav) [reply src/connectedk8s/azext_connectedk8s/custom.py:459]: Removed additional line

File: src/connectedk8s/setup.py (hunk 2 above)
Comment (pasankav) [reply src/connectedk8s/setup.py]: As discussed I'm not reverting changes

**Pull request merged**